css/CSS2/borders/border-color-011.xht

```diff
@@ -6,11 +6,12 @@
   <link rel="alternate" href="http://www.hixie.ch/tests/adhoc/css/inheritance/border-color/001.html" type="text/html"/>
   <link rel="alternate" href="http://www.hixie.ch/tests/adhoc/css/inheritance/border-color/001.xml" type="application/xhtml+xml"/>
   <link rel="help" href="http://www.w3.org/TR/CSS21/box.html#border-color-properties" />
+  <link rel="help" href="https://drafts.csswg.org/css-color/#resolving-color-values" />
   <link rel="match" href="border-color-011-ref.xht" />
-
+  <meta name="assert" content="The initial value of border-color is 'currentColor', which (as of css-color-3) is inherited as a keyword." />
   <style type="text/css">
-   .test { display: block; color: green; border: none; }
-   .test .inner { border-color: inherit; border-style: solid; color: red; }
+   .test { display: block; color: red; border: none; }
+   .test .inner { border-color: inherit; border-style: solid; color: green; }
    .test .inner .text { color: green; }
```

Comment:
The changes are correct, but the `.test .inner .text { color: green; }` statement is no longer useful, and should be removed
```suggestion
```

Reply:
Very true! Done.